### PR TITLE
Editorial: Prefer ssl links over regular http

### PR DESCRIPTION
**Pull request by leobalter**
Ref #330 

cc @anba 

---

There are still some http links that I couldn't find any alternative using https. Mostly for cldr.unicode.org.

### CODE REVIEW

File: meetings/agenda-2018-02-16.md [hunk 1]
```diff
@@ -37,9 +37,9 @@ anba's questions
     1. [Add RelativeDateTimeFormatter.format(FieldPositionIterator)?](https://ssl.icu-project.org/trac/ticket/13256)
         - Dan sent an icu-design email; does anyone want to review?
         - ICU patch wanted
-    1. [RelativeDateTimeFormatter doesn't handle -0 well](http://bugs.icu-project.org/trac/ticket/12936)
+    1. [RelativeDateTimeFormatter doesn't handle -0 well](https://ssl.icu-project.org/trac/ticket/12936)
```
Comment (sffc) [inline meetings/agenda-2018-02-16.md]: Maybe update all the ICU links to go to the new site?

https://unicode-org.atlassian.net/browse/ICU-12936

File: meetings/agenda-2018-02-16.md [hunk 2]
```diff
@@ -37,9 +37,9 @@ anba's questions
...
         - ICU patch wanted
-    1. [TimeZone::getOffset() : add two params to control repeated/skipped wall time](http://bugs.icu-project.org/trac/ticket/13268)
+    1. [TimeZone::getOffset() : add two params to control repeated/skipped wall time](https://ssl.icu-project.org/trac/ticket/13268)
```
Comment (sffc) [inline meetings/agenda-2018-02-16.md]: Another one

File: meetings/agenda-2018-03-16.md [hunk 3]
```diff
@@ -40,7 +40,7 @@ Contact Daniel Ehrenberg (littledan@igalia.com) for the link to the Google Hango
 1. New proposals
     1. [NumberFormat changes](https://github.com/tc39/ecma402/issues/215), including (a) restructuring the spec, (b) minor changes to behavior, (c) support for measure units, and (d) support for scientific and compact notation.
         - Status update for this proposal?
-        - How is the [C API](http://bugs.icu-project.org/trac/ticket/13597) coming (necessary for ChakraCore, at least)
+        - How is the [C API](https://ssl.icu-project.org/trac/ticket/13597) coming (necessary for ChakraCore, at least)
```
Comment (sffc) [inline meetings/agenda-2018-03-16.md]: https://unicode-org.atlassian.net/browse/ICU-.....

There are more.  You can do a global replace if you want to change them.

File: meetings/notes-2018-08-16.md [hunk 4]
```diff
@@ -50,7 +50,7 @@ Agenda and minutes
                 2. ZB: I don't think it does
                 3. JW: The SpiderMonkey implementation doesn't have formatToParts
             24. V8
-                1. FB: The implementation in V8 is complete except formatToParts; maybe we have a workaround that SG would know about. We found a bug in ICU ([ICU-12171](http://unicode-org.atlassian.net/browse/ICU-12171)) and are waiting for a fix to complete
+                1. FB: The implementation in V8 is complete except formatToParts; maybe we have a workaround that SG would know about. We found a bug in ICU ([ICU-12171](https://unicode-org.atlassian.net/browse/ICU-12171)) and are waiting for a fix to complete
```
Comment (sffc) [inline meetings/notes-2018-08-16.md]: This one already has the new link syntax

File: meetings/agenda-2018-03-16.md (hunk 3 above)
Comment (leobalter) [reply meetings/agenda-2018-03-16.md]: sure thing!

**Comment by leobalter** [comment]
@sffc the new patch updates the ICU links

File: meetings/notes-2018-04-20.md [hunk 5]
```diff
@@ -54,7 +54,7 @@ Agenda and notes
         * **Conclusion** - We’ll switch the spec to copy
     4. [PartitionNumberPattern treats -0 as 0](https://github.com/tc39/ecma402/issues/219) (Jack Horton)
         * Shane - we always handled it differently
-            * (Steven: ICU history: [http://bugs.icu-project.org/trac/ticket/8333#comment:14](http://bugs.icu-project.org/trac/ticket/8333#comment:14) [http://bugs.icu-project.org/trac/ticket/13551](http://bugs.icu-project.org/trac/ticket/13551) )
+            * (Steven: ICU history: [https://unicode-org.atlassian.net/browse/ICU-8333#comment:14](https://unicode-org.atlassian.net/browse/ICU-8333#comment:14) [https://unicode-org.atlassian.net/browse/ICU-13551](https://unicode-org.atlassian.net/browse/ICU-13551) )
```
Comment (sffc) [inline meetings/notes-2018-04-20.md]: https://unicode-org.atlassian.net/browse/ICU-8333?focusedCommentId=46005#comment-46005

**Comment by leobalter** [comment]
from this point:

```
git grep 'http\:'
img/ecma-logo.svg:   xmlns:dc="http://purl.org/dc/elements/1.1/"
img/ecma-logo.svg:   xmlns:cc="http://creativecommons.org/ns#"
img/ecma-logo.svg:   xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
img/ecma-logo.svg:   xmlns:svg="http://www.w3.org/2000/svg"
img/ecma-logo.svg:   xmlns="http://www.w3.org/2000/svg"
img/ecma-logo.svg:   xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
img/ecma-logo.svg:   xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
img/ecma-logo.svg:           rdf:resource="http://purl.org/dc/dcmitype/StillImage" /></cc:Work></rdf:RDF></metadata><defs
meetings/notes-2018-03-16.md:            * Eemeli: According to CLDR’s [Core Data for New Locales](http://cldr.unicode.org/index/cldr-spec/minimaldata), ordinal rules are "not required, but are strongly recommended".
meetings/notes-2018-03-16.md:            * PSA [please test the release candidate](http://site.icu-project.org/download/61)
```

File: meetings/notes-2018-04-20.md (hunk 5 above)
Comment (leobalter) [reply meetings/notes-2018-04-20.md]: thanks, this is now fixed!

File: spec/collator.html [hunk 6]
```diff
@@ -252,7 +252,7 @@ <h1>CompareStrings ( _collator_, _x_, _y_ )</h1>
         </p>
 
         <emu-note>
-          It is recommended that the CompareStrings abstract operation be implemented following Unicode Technical Standard 10, Unicode Collation Algorithm (available at <a href="https://unicode.org/reports/tr10/">https://unicode.org/reports/tr10/</a>), using tailorings for the effective locale and collation options of _collator_. It is recommended that implementations use the tailorings provided by the Common Locale Data Repository (available at <a href="http://cldr.unicode.org/">http://cldr.unicode.org/</a>).
+          It is recommended that the CompareStrings abstract operation be implemented following Unicode Technical Standard 10, Unicode Collation Algorithm (available at <a href="https://unicode.org/reports/tr10/">https://unicode.org/reports/tr10/</a>), using tailorings for the effective locale and collation options of _collator_. It is recommended that implementations use the tailorings provided by the Common Locale Data Repository (available at <a href="https://unicode.org/cldr/trac/wiki">https://unicode.org/cldr/trac/wiki</a>).
```
Comment (sffc) [inline spec/collator.html]: This actually is a different site.  Sorry.  The link I suggested is for CLDR Trac, and works for bugs, but not the home page.

I would probably just keep the link the way it is, but if you really want, the following link works:

https://sites.google.com/site/cldr/

**Comment by anba** [comment]
This is great. Thanks!

LGTM with the CLDR links ("cldr.unicode.org") reverted to not point to Trac.

**Comment by littledan** [comment]
This looks good to land as soon as a few of the changes which go to a different site are reverted.

File: spec/collator.html (hunk 6 above)
Comment (leobalter) [reply spec/collator.html]: CLDR needs a https link, perhaps we can convince unicode to use ssl.

**Comment by leobalter** [comment]
Sorry for the long delay, this should be ready to squashed landed.

**Pull request merged**